packages/grapys-react/src/views/GraphRunner.tsx

```diff
@@ -91,62 +103,162 @@ const GraphRunner: React.FC<{ graphData: GraphData }> = ({ graphData }) => {
     }
   }, [graphai, isRunning]);
 
+  const handleKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === 'Enter' && events.length > 0) {
```

Comment:
Will this work fine with Japanese IME? Won't the enter be handled during conversion?

Reply:
I try to test and found no problems. Sorry.